src/applications/mhv-medical-records/tests/reducers/selfEnteredData.unit.spec.js

```diff
@@ -1113,10 +1114,10 @@ describe('convertMilitaryHistory', () => {
         eventTitle: 'Deployment to Iraq',
         eventDate: 'November 10, 2020',
         serviceBranch: 'Army',
+        onboardShip: 'None entered',
```

Comment:
If the you're setting the data to "Yes", why is it being translated to "None entered"?

I think maybe the value in the `pojoObject` is supposed to be "Y" instead of "Yes".